### ISSUE TITLE
Error in "php artisan check:import"

### ISSUE THREAD

**Issue by hmoradian** [imanghafoori1/laravel-microscope#27]
Hello Mr. Iman
When I run "php artisan check:import", This error occorred in end of returned messages:
```
 ErrorException  : Class 'Illuminate\Support\Facades\Input' not found

  at C:\wamp64\www\xxx\vendor\laravel\framework\src\Illuminate\Foundation\AliasLoader.php:80
    76|             return true;
    77|         }
    78|
    79|         if (isset($this->aliases[$alias])) {
  > 80|             return class_alias($this->aliases[$alias], $alias);
    81|         }
    82|     }
    83|
    84|     /**

  Exception trace:

  1   class_alias("Illuminate\Support\Facades\Input", "Input")
      C:\wamp64\www\xxx\vendor\laravel\framework\src\Illuminate\Foundation\AliasLoader.php:80

  2   Illuminate\Foundation\AliasLoader::load("Input")
      [internal]:0

  Please use the argument -v to see more details.
```
I run "composer dump-autoload" command for sometimes too, But as a result, it had no effect!
My laravel version is 6.18

**Comment by insign** [comment]
To be honest, that problems occours here too, but in most commands of this plugin.

**Comment by imanghafoori1** [comment]
Hhhmm,  make sure you use the latest version.
And also provide a sample code which result in error. 

It seems you have a rare syntax in your project that I have not predicted.

**Comment by imanghafoori1** [comment]
AnywAnyway,  bug reports are wellcome and will be fixed in less and 12 hours

**Comment by imanghafoori1** [comment]
@hmoradian the error about `Input` is right.
There is not `Input` facade class in laravel 6.x.
you have to find and remove it from `app.php` file or any other service provider.

**Comment by imanghafoori1** [comment]
@insign Please use the 0.1.41 version, that will help you better report the issue.
It gives you the file name which is not parsable.
So if you give me that file I can find the issue.

**Comment by hmoradian** [comment]
> @hmoradian the error about `Input` is right.
> see this folder: `src/Illuminate/Support/Facades`
> There is not `Input` facade class in laravel 6.x.
> you have to find and remove it from `app.php` file or any other service provider.

Well, I wish your package would send the right message for that

**Comment by imanghafoori1** [comment]
Please do a global text search in your project to find the Facades/Input

And tell me what is the situation.

**Comment by hmoradian** [comment]
I searched globally, but no results were found for "Input" word, in files or content of files, in project or scope(vendor) !

**Comment by imanghafoori1** [comment]
I am sure it is there, search for :
`Illuminate\Foundation\AliasLoader::load("Input")`

do not forget `config/app.php`

**Comment by hmoradian** [comment]
> I am sure it is there, search for :
> `Illuminate\Foundation\AliasLoader::load("Input")`
> 
> do not forget `config/app.php`

I did the following:
1. Remove that line from app.php
2. Change Input::get(...  to  Request::input(... in blade files
2. Run "composer dump-autoload" command.

When run again check:all, I didn't saw that message (Input class not found!)

**Comment by hmoradian** [comment]
For example, I have a method:
```
public static function create($title)
    {
        return new self($title);
    }
```
When ran check:import command, this message shown!:
```
  |*************************************************************************************|
  |    Class does not exist:                                                            |
  |    Vanguard\Support\Sidebar\self  <==== does not exist.                             |
at app\Support\Sidebar\Item.php:37                                                      |
  |*************************************************************************************|
```

**Comment by hmoradian** [comment]
Really, Thank you for Implement my suggestion and idea on Telegram (generate link for open files in editor)

**Comment by imanghafoori1** [comment]
please give me the entire files.
and do not forget to mention the exact version number of laravel-microscope you have problem with.
@hmoradian

**Comment by imanghafoori1** [comment]
The problem with new self should be solved in v0.1.43
You can also contact me on telegram to send files.

**Comment by hmoradian** [comment]
Thanks Mr. Ghafoori
I will send those files with comments and error messages to Telegram for you
